packages/vscode-extension/src/controls/webviewPanel.ts

```diff
@@ -174,10 +177,31 @@ export class WebviewPanel {
 
   private async LoadSampleCollection() {
     await sampleProvider.fetchSampleConfig();
+    const sampleCollection = sampleProvider.SampleCollection;
```

Comment:
What about the experience for CLI?

Reply:
It's covered in this PR: #9841 
Samples not working in current CLI version are excluded.